pkgs/tools/networking/tinyfecvpn/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "tinyfecvpn";
+  version = "master";
```

Comment:
either the version should be "unstable-2019-05-02" or we should use a released version like "20180820.0"

Reply:
I changed it to the stable version